draft-ietf-scitt-architecture.md

```diff
@@ -231,6 +231,11 @@ Receipts are based on Signed Inclusion Proofs, such as those as described in COS
 they can be built on different verifiable data structures, not just binary merkle trees.
 A Receipt consists of a Transparency Service-specific inclusion proof for the Signed Statement, a signature by the Transparency Service of the state of the Append-only Log after the inclusion, and additional metadata (contained in the signature's protected headers) to assist in auditing.
 
+Registered Signed Statement:
+
+: a Signed Statement that has passed the Registration Policy, has been counter-signed by the Transparency Service and has been committed to the Append-only Log.
```

Comment:
```suggestion
: a Signed Statement that has passed the Registration Policy, has been committed to the Append-only Log.
```

Reply:
counter signatures are a different thing.. the definition of registration is inclusion in the log... not signing receipts.

Reply:
I see your point. However, is there a scenario by which a statement is recorded in the log that wasn't counter-signed? A TS must be able to return receipts, that would represent the counter-signature (notarization). Seems we should incorporate that.

Reply:
Technically there can be lots of statements in the log, for which no signed inclusion proof has yet or ever will be produced.

A signed inclusion proof is a kind of counter signature, but we also use the term receipt for it... Less terms is better, if we can avoid using the word counter signature, we should.

Reply:
Russ told us that a countersignature is a countersignature :-) By which he means that it is just that, add more data to it and it becomes its own thing - hence Receipt.

Reply:
I do not think, there is a need of a Registered Signed Statement in the specification. It will be more confusing than doing anything good!.

We have `Signed Statement` and we have `Transparent Statement` 

They are good enough concepts, no need to introduce this new term.

I strongly suggest we drop this Pull Request!

Reply:
Yep, we won't counter sign until a receipt is asked for (if ever is). The VDS is updated waaaaay before that